applications/luci-app-tailscale/htdocs/luci-static/resources/view/tailscale/interface.js

```diff
@@ -0,0 +1,126 @@
+/* SPDX-License-Identifier: GPL-3.0-only
+ *
+ * Copyright (C) 2022 ImmortalWrt.org
+ * Copyright (C) 2024 asvow
+ */
+
+'use strict';
+'require dom';
+'require fs';
+'require poll';
+'require ui';
+'require view';
+
+function formatBytes(bytes, decimals = 2) {
+	if (bytes === 0) return '0 Bytes';
+	const k = 1024;
+	const dm = decimals < 0 ? 0 : decimals;
+	const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+	const i = Math.floor(Math.log(bytes) / Math.log(k));
+	return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
+}
```

Comment:
avoid. See how it's done here
https://github.com/openwrt/luci/blob/1355a6fa253d765140f5571354939d2e93e6d967/modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/29_ports.js#L239-L256